aws_guardduty/lib/botocore/data/eks/2017-11-01/examples-1.json

```diff
@@ -0,0 +1,135 @@
+{
+  "version": "1.0",
+  "examples": {
+    "CreateCluster": [
+      {
+        "input": {
+          "version": "1.10",
+          "name": "prod",
+          "clientRequestToken": "1d2129a1-3d38-460a-9756-e5b91fddb951",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- f1ecfed2-a589-4e16-b3fb-422365199e54 -->